lib/WWW/Spotify.pm

```diff
@@ -159,6 +159,34 @@ has ua => (
     },
 );
 
+has 'response_status' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
+
+has 'response_content_type' => (
+    is      => 'rw',
+    isa     => Str,
+    default => q{}
+);
+
+has 'request_custom_handler' => (
+    is      => 'rw',
```

Comment:
Also wondering why `rw` here?

Reply:
By default this is an empty string, it needs to be set to an anonymous sub in order to make it do anything. See the example in 01-spotify.t

Note this is the exact opposite of spoon feeding which I advocated for the introduction of new methods. Goal with this change is to allow advanced users additional control.

Reply:
> By default this is an empty string, it needs to be set to an anonymous sub in order to make it do anything. See the example in 01-spotify.t

`rw` says that this attribute is mutable after object instantiation. I just wanted to confirm that this is what you had in mind.

> By default this is an empty string

It could also default to `undef` and you could check if it has been set via a predicate. 

```perl
predicate => '_has_custom_request_handler`,
```

That would allow you to declare a type for the attribute.